replicatedhq/ship
# Add Render attribute "Root" to Lifecycle

What I Did
------------
Addresses #294

How I Did it
------------
- Update planner to accept a `root` string, which will create a new `AferoBasePathFs` set to `root`
- Update steps to receive a filesystem as a parameter (`rootFs`)
- Update steps to use `rootFs`

How to verify it
------------
Unit and integration tests have been updated to verify functionality has remained the same. I added an integration test to demonstrate a custom root directory specified in `release.yml`.

Description for the Changelog
------------
- Add Render attribute "Root" to Lifecycle


Picture of a Boat (not required but encouraged)
------------
![](http://i.imgur.com/Ejitu7K.png)
